src/components/MobileHomePageBurgerActive.tsx

```diff
@@ -0,0 +1,88 @@
+import { TypeCategory, headerCategories } from '@/data/categoriesHeader';
+import React from 'react';
+import logoProduct from '@/virus.png';

```

Comment:
Message du linter : 'logoProduct' is declared but its value is never read.ts(6133)

Faut le supprimer vu que c'est pas utilisé

Reply:
Corrigé